apps/ledger-live-mobile/src/locales/en/common.json

```
@@ -462,7 +462,7 @@
       "description": "Try reinstalling the app from the Manager"
     },
     "NotEnoughGas": {
-      "title": "Insufficient ETH for network fee",
+      "title": "The parent account balance is insufficient for network fees",
       "description": "Please send some ETH to your account to pay for ERC20 token transactions."
```

I don't think this `description` is used anywhere (not visible when we get the error and no similar description on LLD 🤷‍♂️)
But just in case, let's update it as well to make it more generic (i.e: not blockchain specific) 

```suggestion
      "description": "Please send some coins to your account to pay for token transactions."
```